### PR TITLE
Separate QPS reporting from balance checking in bank demo.

### PR DESCRIPTION
**Pull request by bdarnell**
Previously we would report QPS at most once per balance check, and the
balance checking transaction is very expensive and requires many
attempts under contention.

Also reduce the default concurrency.

### CODE REVIEW

**Comment by vivekmenezes** [comment]
LGTM

File: examples/kv_bank/main.go [hunk 1]
```diff
@@ -230,6 +220,26 @@ func (bank *Bank) periodicallyCheckBalances(expTotalCash int64) {
...
+	lastNow := time.Now()
+
+	// Wake up periodically to verify the balance and post an update.
+	for range time.NewTicker(time.Second).C {
+		now := time.Now()
+		elapsed := now.Sub(lastNow)
+		numTransfers := atomic.LoadInt32(&bank.numTransfers)
+		newTransfers := numTransfers - lastNumTransfers
+		if newTransfers == 0 {
+			log.Errorf("no progress made")
```
Comment (tamird) [inline examples/kv_bank/main.go]: can you have this report the time elapsed, too?

**Comment by tamird** [comment]
LGTM

File: examples/kv_bank/main.go (hunk 1 above)
Comment (bdarnell) [inline examples/kv_bank/main.go]: The time elapsed here is not interesting: always 1 second plus a tiny fraction.

**Pull request merged**